corehq/apps/app_manager/models.py

```diff
@@ -1533,6 +1533,8 @@ class DetailTab(IndexedSchema):
     """
```

Comment:
The example in this doc string is sort of out of date with the class. It might be better to remove the example all together, as I don't think it adds much (although perhaps an example `nodeset` would be useful).